tekton/rpmbuild/tekton.spec

```diff
@@ -638,6 +638,7 @@ ln -s $PWD/cli-%{version} _build/src/%{import_path}
 export GOPATH="$PWD/_build:%{gopath}"
 export LDFLAGS="${LDFLAGS:-} -X %{import_path}/pkg/cmd/version.clientVersion=%{version}"
 export PATH=$PATH:"%{_builddir}"/bin
+export GO111MODULE=off
```

Comment:
The best way to do this is something like this :

```
%build
export GOPATH="%{_builddir}:%{gopath}"
export PATH=$PATH:"%{_builddir}"/bin
go env -w GO111MODULE=off
go get -u github.com/golang/dep/cmd/dep
GO111MODULE=on go install
```
from what i can read here https://www.reddit.com/r/Fedora/comments/l5bzy0/need_help_creating_a_package_in_copr/

Reply:
Updated according to link